### PR TITLE
Convert ints to vars in GCC first argument for Choco

### PR DESCRIPTION
**Pull request by Dimosts**
Fixes issue #632 

Initial plan was to fix together with the indomain crash from #635, but that issue will be fixed outside of the choco interface.

I also checked other globals in choco if a similar fix is needed, it seems that we have covered similar cases in other globals already.

very specific bug so not sure if test is needed.

### CODE REVIEW

File: cpmpy/solvers/choco.py [hunk 1]
```diff
@@ -592,7 +592,7 @@ def _get_constraint(self, cpm_expr):
                 return self.chc_model.int_value_precede_chain(self._to_vars(cpm_expr.args[0]), cpm_expr.args[1])
             elif cpm_expr.name == "gcc":
                 vars, vals, occ = cpm_expr.args
-                return self.chc_model.global_cardinality(*self.solver_vars([vars, vals]), self._to_vars(occ), cpm_expr.closed)
+                return self.chc_model.global_cardinality(self._to_vars(vars), vals, self._to_vars(occ), cpm_expr.closed)
```
Comment (IgnaceBleukx) [inline cpmpy/solvers/choco.py:595]: The solver_vars also converts numpy to int, so you need to keep that part for `vals` : )

**Review by IgnaceBleukx — APPROVED** [review]
We should probably figure out a more structural way for testing our globals on all solvers with differnent argument types, to prevent these edge-cases in the future. WIll open a discussion issue for it.
